src/main/java/uk/gov/hmcts/reform/sendletter/tasks/MarkLettersPostedTask.java

```diff
@@ -69,7 +69,15 @@ public void run() {
                 .map(parser::parse)
                 .forEach(parsedReport -> {
                     insights.trackPrintReportReceived(parsedReport);
-                    parsedReport.statuses.forEach(this::markAsPosted);
+                    logger.info(
+                        "Updating letters from report {}. Letter count: {}",
+                        parsedReport.path,
+                        parsedReport.statuses.size()
+                    );
```

Comment:
this duplicates previous tracking via insights. maybe we should just use one?

Reply:
I assumed the custom events are for different purpose.
This new info entry makes it easy to see what happened step by step in MarkLettersPostedTask by running a simple query.

Reply:
to my knowledge no1 yet exploiting the custom event tracked via insights. might as well just use info trace alone - there's no need to duplicate data